config.json

```diff
@@ -1,6 +1,11 @@
 {
 	"jqueryUi": [
 		{
+			"version": "origin/master",
+			"dependsOn": "jQuery1.7+",
+			"label": "preview"
```

Comment:
I'm considering deploying this live, replacing version and label once we've got the beta out.